src/ORM/Connect/PDOQuery.php

```diff
@@ -34,17 +120,39 @@ public function __construct(PDOStatement $statement)
 
     public function seek($row)
     {
+        // seek is not available for fetch row
+        if ($this->fetchMedthod === static::FETCH_METHOD_ROW) {
+            return [];
```

Comment:
I believe that throwing an Exception here would actually be more convenient for a developer. Debugging why your row contains no values when you don't realise that seek shouldn't be used with this fetch method would be a footgun. This goes for the other "unsupported" API.